src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/dialogs/layoutconfig.html

```diff
@@ -33,17 +33,23 @@
                                 <div class="tb" style="height: 70px; border-width: 2px; padding: 2px">
                                     <div class="tr">
 
-                                        <a class="td uSky-templates-column"
+                                        <button class="btn-reset td uSky-templates-column"
```

Comment:
Hey @BatJan should this button also have the type attribute?

Reply:
@nathanwoulfe Ah good catch - Yes I think it's better to be explicit even though it works fine without it. But I'll add it here too 👍